src/applications/vaos/covid-19-vaccine/components/SecondDosePage.jsx

```diff
@@ -22,11 +23,18 @@ export default function SecondDosePage() {
     shallowEqual,
   );
   const pageChangeInProgress = useSelector(selectPageChangeInProgress);
+  const featureUseBrowserTimezone = useSelector(
+    selectFeatureUseBrowserTimezone,
+  );
+
   const history = useHistory();
   const dispatch = useDispatch();
   const { date1, vaFacility } = data;
   const selectedDate = parseISO(date1[0]);
-  const timezone = getTimezoneByFacilityId(vaFacility);
+  const timezone = getTimezoneByFacilityId(
```

Comment:
Hmmm now I think about it, aren't facility information always available in the scheduling flow? The user must have selected a facility with a valid ID by the time they get to this page right? If so, this change isn't technically necessary but I guess it'll no op so it's also safe to include.

Reply:
@JunTaoLuo - That is correct

Reply:
Yes, but since the function is exported, you have to check.